src/core/interfaces.ts

```diff
@@ -46,4 +46,12 @@ module Plottable {
     topLeft: Point;
     bottomRight: Point;
   };
+
+  export interface Entity<C extends Component> {
+    datum: any;
+    index: number;
+    position: Point;
+    selection: d3.Selection<any>;
+    component: C;
```

Comment:
That was just a first read impression. Maybe it makes sense. Maybe `position` should be renamed to `coordinate`. I just think the two sound too similar to be able to tell the difference at a first glance